libafl/src/corpus/mod.rs

```diff
@@ -36,7 +36,7 @@ use crate::{inputs::UsesInput, Error};
 /// An abstraction for the index that identify a testcase in the corpus
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, PartialOrd, Ord, Serialize, Deserialize)]
 #[repr(transparent)]
-pub struct CorpusId(pub(crate) usize);
+pub struct CorpusId(pub usize); // MUST BE PUB -- downstream Corpus/Input implementations need it
```

Comment:
Why not add a nicley-named `fn` that does this? Feels like ppl can shoot themselves in the foot here

Reply:
Looking into it, this change is actually not necessary :sweat_smile: